src/Main.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import Menu from './components/Menu/Menu';
+import { getAllGoods } from './request';
+
+export class Main extends Component {
+      constructor(props) {
+    super(props);
+    this.state = {
+      currentProducts: {},
+      category: 'Закуски'
```

Comment:
Move to constant.

Reply:
Create folder "constants" inside this folder create files that contains some constants.

Reply:
Done